client/packages/system/src/Item/Components/PackVariant/PackVariantQuantityCell.tsx

```diff
@@ -23,10 +26,19 @@ export const PackVariantQuantityCell =
 
     const quantity = getQuantity(rowData);
     const packQuantity = numberOfPacksFromQuantity(quantity);
+    const hasMoreThanTwoDp =
+      packQuantity.toString().split('.')[1]?.length ?? 0 > 2;
+    const roundedPackQuantity = NumUtils.round(packQuantity, 2);
```

Comment:
would useFormatNumber.dumber work here ?

Reply:
Shouldn't use in callback 👀

Reply:
Hmm, we use `usePackVaraint` hook here, would that be problematic ?